core/src/main/java/com/linecorp/armeria/server/CatchAllPathMapping.java

```diff
@@ -18,17 +18,17 @@
 
 import static com.linecorp.armeria.internal.RouteUtil.ROOT_LOGGER_NAME;
 
-import java.util.Optional;
+import java.util.List;
 import java.util.Set;
 
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
 
 final class CatchAllPathMapping extends AbstractPathMapping {
 
-    static final CatchAllPathMapping INSTANCE = new CatchAllPathMapping();
+    static final CatchAllPathMapping instance = new CatchAllPathMapping();
```

Comment:
Just to check, is this intentional? I think we've so far used all-caps for `INSTANCE`.

Reply:
Yeah, we have used that. But it's technically not a constant. It's not public, so I changed it without much worry. Throughout the whole our code, we use two different conventions for the static final references. I want to change them all, but some of them are public so I couldn't.

Reply:
Do you think we need to keep the variable using underscores with uppercase? 🤔

Reply:
Changed it back to `INSTANCE`. :-)